lib/mixin/vpp.py

```diff
@@ -224,3 +224,15 @@ def check_color_props(self):
     metrics2.check(**params)
 
   check_range  = check_color_props
+
+  def check_drawbox(self):
+    owidth, oheight = self.width, self.height
+
+    metrics2.check(
+      metric = dict(type = "filesize"),
+      filetest = self.decoded, width = owidth, height = oheight,
+      frames = self.frames, format = self.format)
+
+    params = vars(self).copy()
+    params.update(width = owidth, height = oheight)
+    metrics2.check(**params)
```

Comment:
Copy/update is unnecessary... nothing changed.  How about call `self.check_default()` instead?

Reply:
Updated, thx